internal/service/federatedsettingsidentityprovider/resource_federated_settings_identity_provider.go

```diff
@@ -7,6 +7,9 @@ import (
 	"net/http"
 	"regexp"
 
+	oldAdmin "go.mongodb.org/atlas-sdk/v20231001002/admin"
+	admin "go.mongodb.org/atlas-sdk/v20231115002/admin"
```

Comment:
do you need admin package alias here?

Reply:
Yes it's needed for some structs

Reply:
i mean if the alias is needed or you can just import "go.mongodb.org/atlas-sdk/v20231115002/admin"

Reply:
oh I understand, I put it to be more explicit, but yes, `admin` alias is not needed. Once we clean up we can leave it as it was

Reply:
i would keep the current admin without alias as in all other files to avoid confusion, and only use alias in old one

Reply:
Changed